module/x/gravity/abci.go

```diff
@@ -281,7 +281,7 @@ func cleanupTimedOutBatchTxs(ctx sdk.Context, k keeper.Keeper) {
 		btx, _ := otx.(*types.BatchTx)
 
 		if btx.Timeout < ethereumHeight {
-			// we keep the data around for slashing in case the timeout was due to a lack of signatures
+			// we keep the data around for slashing in case the timeout was due to a lack of signatures.
 			k.SetCompletedOutgoingTx(ctx, btx)
 			k.CancelBatchTx(ctx, btx)
```

Comment:
If we're using a bundled `Complete` function, wouldn't we want to do that here as well?